### PR TITLE
Changing mkdocs

### PR DESCRIPTION
**Pull request by joeyouss**
Changed each page of mkdocs to make howdoi's documentation better

### CODE REVIEW

File: docs/contributing_to_howdoi.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#### 3 - Making a Pull request (PR)
+
+- After you have worked on the issue and fixed it, we need to merge it from your forked repository into the howdoi repository. This is done by making a PR.
+- You can search
+  ```
+  howdoi create a pull request on Github
+  ```
+  in your command line and follow the steps written in it.
+- Each PR made should pass all the ) tests and should not have any flake8 or pylint errors or warnings. Hence, testing and linting are important before making a PR. You can now run `python setup.py` lint which will run pylint and flake8.:
```
Comment (gleitz) [inline docs/contributing_to_howdoi.md:25]: The wording is strange here

File: docs/extension_dev.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+## Extension development
+
+You can integrate howdoi as a code editor extension. For this, you need to edit the files within the `extension/code-editor-integration` folder.
+To improve the Visual Studio Code howdoi extension, edit the files within the extension/vscode-ext folder and republish the extension.
```
Comment (gleitz) [inline docs/extension_dev.md:4]: Can you link to these two folders on github?

File: docs/extension_dev.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+#### How to integrate howdoi as a code editor extension?
+
+1. The Code Editor Integration plug-in runs when you call
+   ` runHowdoi` function which takes in a user’s query(type = string). The parameter is encapsulated by a single line comment and is formatted as : `// howdoi query`
+
+   `runHowdoi` function returns and Object which looks like :
+
+```
+{
+Question: string
```
Comment (gleitz) [inline docs/extension_dev.md:15]: two space indent here to make it more readable

File: docs/extension_dev.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Answer : string[]
+Link : string[]
+}
+```
+
+2. What are these object values?
+
+- question = it contains the user’s query encapsulated by a single line comment
```
Comment (gleitz) [inline docs/extension_dev.md:23]: Pick equals signs or colons here, but not both

File: docs/extension_dev.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+- Import the `plugin.ts` file into your main file.
+- Call the `runHowdoi` function.
+
+#### Visual Code Extension development
+
+To begin the development for Visual Studio Code extension, install all the necessary packages:
+
+`Npm install`
```
Comment (gleitz) [inline docs/extension_dev.md:57]: lowercase npm here and below

File: docs/howdoi_advanced_usage.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  ```
+  $ h format date bash
+  ```
+
+- **Other uses and aliases** - You can also search other StackExchange properties for answers.
+
+  Example:
+
+  ```
+  $ HOWDOI_URL=cooking.stackexchange.com
```
Comment (gleitz) [inline docs/howdoi_advanced_usage.md:26]: I think you can put these two lines together into a single block quote here and also `hcook` below.

File: howdoi/howdoi.py [hunk 7]
```diff
@@ -583,6 +583,7 @@ def _parse_cmd(args, res):
 
 
 def howdoi(raw_query):
+    # print("checking")
```
Comment (gleitz) [inline howdoi/howdoi.py:586]: Remove

**Comment by gleitz** [comment]
Are all these comments resolved? If so I'll do a final review

**Comment by joeyouss** [comment]
Yes @gleitz

**Pull request merged**